docs/topics/eap.md

```diff
@@ -26,4 +26,38 @@ check [our instructions on how to configure your build to support this version](
 
 ## Build details
 
-_No preview versions are currently available._
\ No newline at end of file
+<table>
+    <tr>
+        <th>Build info</th>
+        <th>Build highlights</th>
+        <th>Recommended kotlinx library versions</th>
+    </tr>
+    <tr>
+        <td><strong>1.5.30-M1</strong>
+            <p>Released: <strong>July 19, 2021</strong></p>
+            <p><a href="https://github.com/JetBrains/kotlin/releases/tag/v1.5.30-M1" target="_blank">Release on GitHub</a></p>
+        </td>
+        <td>
+             <ul>
+                <li>Kotlin/Native: support for Apple silicon, improved Kotlin DSL for Cocoapods Gradle plugin, optimized access to object declarations with `const val`s only, and debugger improvements.</li>
+                <li>Kotlin Multiplatform: sharing custom <code>cinterop</code> libraries between platforms, new default publishing setup for Android artifacts, and XCFrameworks support.</li>
+                <li>Kotlin/JS: source maps generation for the IR compiler backend.</li>
+                <li>Swift/Objective-C interop: experimental interoperability with Swift 5.5 async/await, improved singleton mapping</li>
+                <li>Passing by value for more C structs using C interop.</li>
+                <li>Compiler improvements: requiring opt-in on implicit usages of experimental APIs, changes in using opt-in requirement annotations with different targets.</li>
+            </ul>
+            <p>For more details, please refer to the <a href="https://github.com/JetBrains/kotlin/releases/tag/v1.5.30-M1">changelog</a> or <a href="https://blog.jetbrains.com/kotlin/2021/07/kotlin-1-5-30-m1-released/?utm_source=ide&amp;utm_medium=change-notes&amp;utm_campaign=1-5-30-eap">this blog post</a>.</p>
```

Comment:
Perhaps the blog post link should have another `utm_meduim` or not have utm tags at all. Please check with marketing.

Reply:
Thanks, checked and updated.